### PR TITLE
DaringDevelopmentInc.Horizon version 2.9.0

### PR DESCRIPTION
**Pull request by Masamune3210**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DaringDevelopmentInc.Horizon
+PackageVersion: 2.9.0
+PackageLocale: en-US
+Publisher: Daring Development Inc.
+PackageName: Horizon
+License: Copyright (c) Daring Development Inc.. All rights reserved.
```
Comment (ItzLevvie) [inline manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml:9]:
```suggestion
License: Copyright (c) Daring Development Inc. All rights reserved.
```

File: manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DaringDevelopmentInc.Horizon
+PackageVersion: 2.9.0
+PackageLocale: en-US
+Publisher: Daring Development Inc.
+PackageName: Horizon
+License: Copyright (c) Daring Development Inc.. All rights reserved.
+ShortDescription: Horizon
```
Comment (ItzLevvie) [inline manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml:10]:
```
ShortDescription: Horizon
```
Please write a better description for what this application is about.

File: manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: DaringDevelopmentInc.Horizon
+PackageVersion: 2.9.0
+Installers:
+- Architecture: x86
+  InstallerType: burn
+  InstallerUrl: https://www.wemod.com/horizon/download
+  InstallerSha256: 0F832DD0FCE1413B4C785B718D132D39ECAD00E410824E734B5325083082AA03
```
Comment (ItzLevvie) [inline manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.installer.yaml:10]:
```suggestion
  InstallerSha256: 0F832DD0FCE1413B4C785B718D132D39ECAD00E410824E734B5325083082AA03
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml (hunk 2 above)
Comment (Masamune3210) [reply manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml]: Sorry, still really new to this, thought I had already put a better one in

File: manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DaringDevelopmentInc.Horizon
+PackageVersion: 2.9.0
+PackageLocale: en-US
+Publisher: Daring Development Inc.
+PackageName: Horizon
+License: Copyright (c) Daring Development Inc. All rights reserved.
+ShortDescription: Horizon Modding Tools and Mod Manager
```
Comment (ItzLevvie) [inline manifests/d/DaringDevelopmentInc/Horizon/2.9.0/DaringDevelopmentInc.Horizon.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Horizon is the world’s most powerful Xbox 360 modding tool.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**